manage_projects.rb

```diff
@@ -180,10 +176,12 @@ def update_regions(project)
             else
               continue = true
             end
+          else
+            continue = true
           end
         end
         regions << region
-        metadata[:regions] = regions.uniq
+        metadata[:regions] = regions.uniq.reject { |region| region.strip.empty? }
```

Comment:
You're right - was thinking that it would handle if there was already a region that was blank, but that shouldn't now be possible (unless say the user badly manipulates things in `irb`).